content/docs/components/hyperparameter-tuning/katib-config.md

```diff
@@ -114,7 +114,13 @@ All of these settings except **`image`** can be omitted. If you don't specify an
 
```

Comment:
I think we should merge this PR after we update Katib Manifests images with the latest images.
/cc @johnugeorge